src/app/shared/model/audio-version.model.ts

```diff
@@ -127,7 +127,7 @@ export class AudioVersionModel extends BaseModel {
   }
 
   changed(field?: string | string[], includeRelations = true): boolean {
-    if (this.isNew && this.files.length === 0) {
+    if (this.isNew && this.files.length === 0 && this.explicit === undefined) {
```

Comment:
Hmm... having to manually check for explicit here seems like a symptom of something else.

Maybe try giving it a default value of `""` (empty string) at the top of this file?  Could also mess around with the decode/encode functions, to allow a blank value rather than defaulting to 'Clean'.

Or maybe if clean really is the default, the top should be `public explicit = 'Clean'`?